dev/furuno/nnd-parser.py

```
@@ -0,0 +1,545 @@
+def decode_mode0(src, n):
+    """Raw uncompressed."""
+    return list(src[:n]), n
+
+
+def decode_mode1(src, n):
+    """Intra-spoke RLE, 7-bit pixels."""
+    dst = []
+    last = 0
+    s = 0
+    while len(dst) < n and s < len(src):
+        b = src[s]; s += 1
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)